agent_workflows/prompts/llama3-prompt_engineering-final_chain.yaml

```diff
@@ -11,7 +11,7 @@ template: |
     You are an assistant for question-answering tasks.
     Use the following generation to provide a final answer to the original question.  If you cannot answer based on the generation provided to you, say "I apologize, but I cannot provide the final answer due to a lack of grounding in documents or an ability to generate and run code that is required to solve the task".  Please do not comment on how to obtain further information about partially answered questions.  If there is specific advice that could help the user find information from the provided generation, do provide all of this advice as part of your answer/apology.<|eot_id|><|start_header_id|>user<|end_header_id|>
     Original Question: {question} 
-    Generation: {generation} 
```

Comment:
call_rag methods expect generation.  Here is an example:

https://github.com/sambanova/ai-starter-kit/blob/7e9c9477c2235d444b1b59954a3841c1ccb6df8d/utils/agents/static_RAG_with_coding.py#L246